lib/net.js

```diff
@@ -1389,7 +1389,7 @@ Server.prototype.listen = function() {
 };
 
 function lookupAndListen(self, port, address, backlog, exclusive) {
-  require('dns').lookup(address, function(err, ip, addressType) {
+  require('dns').lookup(address, function emitDnsLookup(err, ip, addressType) {
```

Comment:
I put it on 6ce4862ca1cb500abd6b2ec54ae8f5e1344cad61 probably now LGTY (looks good to you ) 😄